### ISSUE TITLE
Deploy app to heroku

### ISSUE THREAD

**Issue by rajvaibhavdubey** [fossasia/yaydoc#447]
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

Temporarily app should be deployed to heroku.

**Comment by Dishebh** [comment]
@rajvaibhavdubey if it is still up for grabs, can I take it?

**Comment by mariobehling** [comment]
@Dishebh Yes, please go ahead. Please deploy app automatically on each merged PR to Heroku on each merged PR.

**Comment by pranavkarthik10** [comment]
@mariobehling I successfully setup automatic deployment of yaydoc from my fork of the GitHub repository to heroku. It is currently available at yaydoc-gci.herokuapp.com. Because heroku requires OAuth access from GitHub for auto-deployment, it is currently limited to only merged PR's and commits from my fork of the repository. If I can get collaborator access to the main repository, I can set it up for fossasia's original repo.
